src/routes/collection.ts

```diff
@@ -44,11 +44,14 @@ export async function getCollection(slug: string) {
 function transformToBrazePayload(response): BrazeCollections {
   const collection = response.data.getCollectionBySlug;
   const stories = collection.stories.map((story) => {
-    return {
+    const res = {
       ...story,
       imageUrl: getResizedImageUrl(story.imageUrl),
       authors: story.authors.map((author) => author.name),
+      shortUrl: story.item.shortUrl,
     };
+    delete res.item;
+    return res;
```

Comment:
nit: I think we can do something like this to avoid the `delete` but doesn't really matter. Also, I think TS would give you a lint error for this:
```
const {item, ...storyWithoutItem} = story;
const res = { ...storyWithoutItem, ... }; 
return res;
```

Reply:
I like adapting that lint rule to suppress it if the variable is preceded by an underscore:
```
const { item: _item, ...storyWithoutItem } = story;
```

or even just:
```
const { item: _, ...storyWithoutItem } = story;
```

It's a common convention for unused variables, and is a lot less clunky than lint rule exceptions.

The memory is already allocated if you're spreading. You get improved performance (the garbage collector gets to clean at its leisure instead of deleting **now**), and you avoid mutation of the original object if it gets shared elsewhere.

Reply:
the item gets shown in the response, would setting `item: _` make sure its removed from the response?

Reply:
done